fire-hose-management-frontend/src/components/Departments/DepartmentOverlay.jsx

```diff
@@ -1,158 +1,207 @@
 import * as i18nIsoCountries from "i18n-iso-countries";
-import React, { useState } from "react";
+import React, {useState} from "react";
```

Comment:
Codacy has a fix for the issue: [Replace `useState` with `·useState·`](https://app.codacy.com/gh/StitZle/fire-hose-complete/pullRequest?prid=9542260)
```suggestion
import React, { useState } from "react";
```